fbw-common/src/systems/navdata/client/backends/Msfs/Mapping.ts

```diff
@@ -166,6 +166,7 @@ export class MsfsMapping {
       longestRunwaySurfaceType: this.mapRunwaySurface(longestRunway[1]?.surface),
       transitionAltitude,
       transitionLevel,
+      magneticVariation: msAirport.magvar,
```

Comment:
This is only available in MSFS2024. Should we fill this with the WMM value in 2020?

Reply:
I would suggest actually not using it... It is often overriden by the airport scenery making it not very useful or actively harmful.